### PR TITLE
Mostly Amiga remakes

### PR DESCRIPTION
**Pull request by wonea**

### CODE REVIEW

File: games.yaml [hunk 1]
```diff
@@ -570,6 +570,13 @@
       repo: https://github.com/lutzroeder/digger
       info: active development, TypeScript, no specified licence
 
+- name: [Bratwurst, http://www.mobygames.com/game/bratwurst]
```
Comment (piranha) [inline games.yaml]: You need to enclose `http://...` address in double quotes to fix the issue.

**Comment by wonea** [comment]
All fixed now, and added a few more games. Think I might switch tracks now, and find PlayStation games instead.

**Comment by wonea** [comment]
@piranha bump! I've found some more games, any chance of getting this merged?

**Comment by piranha** [comment]
oops! why didn't I yet merge that?!

**Pull request merged**